src/sys/resource.rs

```diff
@@ -10,10 +10,14 @@ pub use libc::RLIM_INFINITY;
 use std::mem;
 
 cfg_if! {
-    if #[cfg(all(target_os = "linux", any(target_env = "gnu", target_env = "uclibc")))]{
+    if #[cfg(any(
+        target_os = "hurd",
+        all(target_os = "linux", any(target_env = "gnu", target_env = "uclibc"))
+    ))]{
         use libc::{__rlimit_resource_t, rlimit};
     } else if #[cfg(any(
         target_os = "freebsd",
+        target_os = "hurd",
```

Comment:
Here you have hurd on both sides of the `cfg-if!`.  Is that really intended?

Reply:
oh, right, that's because I had fixed the compilation one way at some point, and another way at another point. In the end it's indeed the first case that matters, fixed so.